### DISCUSSION TITLE
Button text disappears

### DISCUSSION THREAD

**Discussion by ripplethor** [bigskysoftware/htmx#2765] [General]
I have been going around in circles trying to figure out a way to stop my button text disappearing.

`<button  hx-post="/?opp.test" hx-preserve="true" hx-swap="none">Continue</button>`

I would have thought that having an hx-swap of none, would leave the text 'Continue' untouched and not replaced, but as soon as the button is clicked the word disappears.

**Comment by Telroshan** [comment]
Hey, that's weird, indeed `hx-swap="none"` should be enough, as can be seen on [this JSFiddle](https://jsfiddle.net/ys4arowt/)

Would you mind sharing a reproducible example, to be able to investigate that issue?
